src/language/concurrency.md

```diff
@@ -16,90 +19,99 @@ prevpage:
   }
 </style>
 
-Dart supports concurrent programming with async-await, isolates, and
-classes such as `Future` and `Stream`.
-This page gives an overview of async-await, `Future`, and `Stream`,
-but it's mostly about isolates.
-
-Within an app, all Dart code runs in an _isolate._
-Each Dart isolate has a single thread of execution and
-shares no mutable objects with other isolates.
-To communicate with each other,
-isolates use message passing.
-Many Dart apps use only one isolate, the _main isolate_.
-You can create additional isolates to enable
-parallel code execution on multiple processor cores.
-
-Although Dart's isolate model is built with underlying primitives
-such as processes and threads
-that the operating system provides,
-the Dart VM's use of these primitives
-is an implementation detail that this page doesn't discuss.
+Concurrent programming in Dart refers to both asynchronous APIs, like `Future`
```

Comment:
Somewhere at the start, I think there should be a clear purpose statement for the page, e.g. "This page contains a conceptual overview of Dart's options for concurrent programming" or something like that. "Conceptual overview" is also not the clearest statement, but something that tells the reader "this is reading/understanding/foundational info, not doing/practical/instructional info" before they dive in